tools/graph_bag/scripts/utilities.py

```diff
@@ -77,6 +77,7 @@ def load_dataframe(files):
 
 
 def run_command_and_save_output(command, output_filename):
+    print(command)
```

Comment:
did you mean to leave this?

Reply:
Oh yeah I did, but I'll make it optional and pass a bool for printing.  I found it helpful to have the command being run printed to the file but I can see how other times people wouldn't want that

Reply:
The logging module may be a good way to handle that.